system/core/pages.php

```diff
@@ -195,6 +195,15 @@ function loadPage(RequestContext $O_O) {
 					exit;
 				}
 				break;
+        // hier invoegen
+        case HyphaRequest::HYPHA_SYSTEM_PAGE_SEARCH: //bz search
+          $hyphaHtml->writeToElement('pagetitle',__('searchresults')); //add searchtitle to languages
```

Comment:
This sets 'pagetitle', but this element does not seem to be defined anywhere? At least no in my git master version?